refract-spec.md

```diff
@@ -36,18 +36,21 @@ The Refract Element contains four properties: `element`, `meta`, `attributes`, a
           - `ref` (Link) - Link to referenced element or type
           - `class` (array[string]) - Array of classifications for given element
           - `prefix` (string) - Prefix in which element MAY be found
+          - `name` (enum) - For use when an element is used as a property of an Object Element
```

Comment:
@smizell can you please make this more generic? So other elements can make a use of it? Thanks.